src/Provider/PaymentServiceProvider.php

```diff
@@ -14,7 +14,7 @@ class PaymentServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        $this->loadViewsFrom(__DIR__.'/resources/views', 'shetabitPayment');
+        $this->loadViewsFrom(__DIR__.'/../../resources/views', 'shetabitPayment');
```

Comment:
what would happen if we have not published views to resources path?

Reply:
I just fixed the bug, if you wouldn't have blade file you can remove blade register codes from this file

Reply:
i mean what would happen if a user has not published views using `vendor:publish` ? if we load views from `resources/views` path, it can cause fetal errors.

Reply:
No, I haven't published views and I don't get error, I test it after publishing views and again nothing happened

Reply:
which drivers do you use?

Reply:
i think your solution is not a good one. by default, package views must be loaded from it self directory. loading views from project `resources/views` path it not a good solution.

how ever let me see some famous packages and check them. if they use this solution too, i will merge your pull request.

Reply:
oh i miss understood! you load it from the package. the address was wrong.

i merge it now.

thanks for your contribution.

Reply:
It is my pleasure